### PR TITLE
Upgrade Sails - circle to Node 4 

### PR DESCRIPTION
**Pull request by rargulati**
Have `circle.yml` now call the makefile to install this dependency. This pattern emerged from work in `shyp_api`.

### CODE REVIEW

**Comment by kevinburkeshyp** [comment]
You need to rebase against master to get the `microtime` change.

**Comment by rargulati** [comment]
Yep

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,18 @@
 .PHONY: test install clean shrinkwrap
 
+NODE_VERSION = v4.4.0
```
Comment (kevinburkeshyp) [inline Makefile]: We don't need this anymore, oops :(

**Comment by kevinburkeshyp** [comment]
LGTM

**Pull request merged**